### DISCUSSION TITLE
How can i hide selected item from list box

### DISCUSSION THREAD

**Discussion by RajaTheKing826** [adobe/react-spectrum#7061] [Q&A] [answered]
I have an use case, where i have to show my selected item text in input only and i have to remove that item from list box. How can i do this?

**Comment by reidbarber** [comment, marked as answer]
Is this for ComboBox? If so, you could utilize a controlled custom value where you set that value when an item is selected and remove that item from the collection. You would probably want to add that back to the collection when the value gets changed/cleared though.

**Reply by RajaTheKing826** [reply]
Got it
